coordinator/src/node.rs

```diff
@@ -60,20 +59,18 @@ impl Node {
             .filter(|channel| channel.is_usable && channel.counterparty.node_id == *trader)
             .collect::<Vec<_>>();
 
-        // should be exactly 1
+        if useable_channels.len() > 1 {
+            tracing::warn!(%trader, "Found more than one useable channel with trader");
+        }
         !useable_channels.is_empty()
     }
 
     pub async fn trade(&self, trade_params: &TradeParams) -> Result<()> {
         match self.decide_trade_action(&trade_params.pubkey)? {
             TradeAction::Open => self.open_position(trade_params).await?,
             TradeAction::Close(channel_id) => {
-                let trader_pk = trade_params.pubkey;
-                tracing::info!(
-                    order_id = %trade_params.filled_with.order_id,
-                    %trader_pk,
-                    "Closing position"
-                );
+                let peer_id = trade_params.pubkey;
+                tracing::info!(?trade_params, ?channel_id, ?peer_id, "Closing position");
```

Comment:
Don't need to print as `Debug` for `channel_id` and `peer_id` I think.

Actually, for the `channel_id` we usually convert to hex before printing:

```rust
channel_id = %hex::encode(channel_id)
```

Reply:
it would have been heaps better to just have a `ChannelId` struct that's `Display` is hex encoding.